addons/bus/static/src/debug/bus_logs_menu_item.js

```diff
@@ -0,0 +1,39 @@
+import { Component, useRef, useState } from "@odoo/owl";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+/**
+ * @typedef {Object} Props
+ * @extends {Component<Props, Env>}
+ */
+export class BusLogsMenuItem extends Component {
+    static template = "bus.BusLogsMenuItem";
+    static components = { DropdownItem };
+
+    setup() {
+        this.busLogsService = useState(useService("bus.logs_service"));
```

Comment:
No need to `useState()` around `useService()`, this is automatic with stateful/reactive service.